backend/src/v5/utils/helper/yup.js

```diff
@@ -117,7 +118,8 @@ YupHelper.types.dateInThePast = YupHelper.types.date.test(('date-in-the-past', '
 }));
 
 const imageValidityTests = (yupType, isNullable) => yupType.test('image-validity-test', 'Image is not valid', async (value, { createError, originalValue }) => {
-	if (isUUIDString(originalValue)) {
+	const isImageRef = isUUIDString(originalValue) || (isUUID(originalValue) && !isBuffer(originalValue));
```

Comment:
why do you need the extra bits when we didn't need it before? 🤔 

I think the check should be done in the middleware in which case it should also be in string form?

and surely if it's a UUIDString it'll never be a buffer value?

Reply:
its the same reason as below :/ the first time this is run its actually a UUID string so it returns true. The second time though its a UUID and it proceeds to extract the extention which is undefined and it returns error. 

With this fix, if I pass a ref it will return true the first time cause of the first OR clause and the second time because of the second clause while a base 64 string is not caught any of the times and goes to check the extention

Reply:
ok, why do you need to check !isBuffer? can a buffer be a buffer and UUID at the same time? surely that's just a UUID if that's the case?